.github/workflows/pr-and-push.yml

```diff
@@ -17,3 +17,15 @@ jobs:
       contents: read
     with:
       ref: ${{ github.event.pull_request.head.sha }}
+
+  check-api:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+    steps:
+    - uses: actions/checkout@v5
+      with:
+        fetch-depth: 0  # We the need the full Git history.
+    - uses: astral-sh/setup-uv@v6
+      # The following command will compare current changes to latest tag.
+    - run: uvx griffe check --search src --format github strands
```

Comment:
From the docs:

> By default, Griffe will compare the current code to the latest tag

I think we'd want to compare this to the base commit of the PR rather than the latest?